BUILDING.md

```diff
@@ -283,6 +283,12 @@ may reduce build time. For more information, see the
 The above requires that `python` resolves to a supported version of
 Python. See [Prerequisites](#prerequisites).
 
+A stale build can lead to issues, for example a `file not found` error.
+To fix these, please try
```

Comment:
Move this to the end of the previous paragraph rather than having it be a new paragraph referring to "these" from the previous paragraph.